packages/core/types/src/workflow/order/create-shipment.ts

```diff
@@ -0,0 +1,17 @@
+import { BigNumberInput } from "../../totals"
```

Comment:
suggestion: Maybe we can start moving the workflow types that are located in src/order, no necessarely in that pr though